packages/@aws-cdk/aws-ecs/test/fargate/integ.firelens-cloudwatch.ts

```diff
@@ -0,0 +1,44 @@
+import * as ec2 from '@aws-cdk/aws-ec2';
```

Comment:
Can you also make sure that there's an integ test which uses the S3 configuration file type? I'm worried that the permissions might be inadequate and I want to make sure that the newly created container can actually get its configuration options. 

One approach to this could be to store the config file in the integ directory, then upload it directly from there to a newly created s3 bucket, then create the rest of the integ test resources.

Reply:
yes, added the integ test: ec2/integ.firelens-s3-config.ts